deployments/gac/common-aws.nix

```diff
@@ -0,0 +1,20 @@
+{ config, ... }:
+
+{
+  ## Universal resource logic:
+  resources.ec2KeyPairs."cardano-keypair-${config.node.org}-${config.node.region}" = {
+    inherit (config.node) region accessKeyId;
+  };
+
+  resources.ec2SecurityGroups = {
+    "allow-deployer-ssh-${config.node.region}-${config.node.org}" = {
```

Comment:
with the latest changes in the master branch, you can also do:
`require = [ ./security-groups/allow-deployer-ssh.nix ];`
but that relies on `io` having done `nixops set-args --arg globals 'import ./globals.nix ....'`

Reply:
We could do some gymnastics to get around `io`'s `globals` but wouldn't it just make things more difficult to reason about? @deepfire thoughts?